### DISCUSSION TITLE
Heltec v3 firmware 2.5.11 - configuration doesn't fully restore

### DISCUSSION THREAD

**Discussion by overlord23** [meshtastic/firmware#5331] [Troubleshooting]
Hi folks

I came across an issue while restoring the configuration on my Heltec V3, here's the steps applied:

- issued nodedb reset and factory reset
- configured node and exported config with Meshtastic CLI
- issued nodedb reset and factory reset again
- restored config from Meshtastic CLI - serial output printed out `ERROR | 22:25:38 593 Unknown config type 10`
- exported config from Meshtastic CLI and compared with the previously exported yml

The following parameters included in the configuration file didn't restore:
- autoScreenCarouselSecs:
- txPower
- adcMultiplierOverride
- root (lora region)

Am I missing something or this could be a bug?

**Environment:** 

Board: Heltec V3
Firmware: 2.5.11
Meshtastic CLI: 2.5.4

**Comment by fifieldt** [comment]
Just for reference -- "ERROR | 22:25:38 593 Unknown config type 10" is not a problem.

You are right that the others should restore

**Comment by fifieldt** [comment]
Do you have full logs available?

**Reply by overlord23** [reply]
Here are the serial log collected while uploading the configuration and the debug log after uploading.
[debug_aftrestore.txt](https://github.com/user-attachments/files/17740013/debug_aftrestore.txt)
[serial_log_cfgrestore.txt](https://github.com/user-attachments/files/17740014/serial_log_cfgrestore.txt)

**Comment by overlord23** [comment]
I managed to get a Heltec Wireless Stick Lite V3 and replicated the previous steps, the issue happens also with wsl firmware

**Comment by lucaskatayama** [comment]
I get something like this...

1. Export config
2. Change `mqtt.enable` from `true` to `false`
3. Apply with `--configure <YAML> --port /dev/<PORT>`
Doesn't change anything... 

BUT 
Using bluetooth (`--ble <name>`), it works... 
It only happens with serial port ....
